src/shipping/shipping-country-reducer.spec.ts

```diff
@@ -35,7 +35,8 @@ describe('shippingCountryReducer()', () => {
         const action: LoadShippingCountriesAction = {
             type: ShippingCountryActionType.LoadShippingCountriesSucceeded,
             payload: [
-                { code: 'JP', name: 'Japan', hasPostalCodes: false, subdivisions: [] },
+                { code: 'JP', name: 'Japan', hasPostalCodes: false, subdivisions: [], isRequiresStateExperimentOn: false,
```

Comment:
I'm thinking that the SDK should probably unaware of the experiment. Probably the experiment should simply be applied at an API level. If it's false, then `requiresState` should always be true whenever there are subdivisions.

Reply:
The changes are now complete. Removed all dependency from isRequriesStateExperimentOn